e2e/test_bootstrap_sdist_only.sh

```diff
@@ -10,6 +10,9 @@
 SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 source "$SCRIPTDIR/common.sh"
 
+DIST=stevedore
```

Comment:
Should this be matplotlib?

Reply:
Whew, ok, I don't think we want that. I was looking for a way for the test here to show that the settings are passed through, but maybe that's not needed in an e2e test, since you have the unit tests.